embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/crumbs/EmbraceBreadcrumbService.kt

```diff
@@ -223,6 +223,9 @@ internal class EmbraceBreadcrumbService(
      * Close all open fragments when the activity closes
      */
     override fun onViewClose(activity: Activity) {
+        if (!configService.breadcrumbBehavior.isActivityBreadcrumbCaptureEnabled()) {
```

Comment:
An unrelated change got included.

Reply:
I added this back in as it was previously included as a check: https://github.com/embrace-io/embrace-android-sdk/pull/424/files#diff-1756a1f692be8ae6c6266377829a385a0de54b3ca51ea08bcd9c8159688883fdR225